src/content/guides/author-libraries.md

```diff
@@ -174,6 +176,30 @@ module.exports = {
 };
 ```
 
+## Base Configuration with source map 
+ Source maps is a useful debugging tool to enable you to view where the original code was
+ For more information about this and to see code examples please refer to:
+ https://webpack.js.org/configuration/devtool/
```

Comment:
Instead of providing urls, consider using some text with those as links. That way people will know where they are going.

Reply:
I separated the two links and gave them a line explaining where they lead